index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+  </head>
+
+  <body>
+  <header>  
+    <nav class="header" id="logo">
+      <h3 class="poppins"><a href="#">My Logo</a></h3>
+      <img src="./assets/hamburger menu.png">
+    </nav>
+  </header>
+  
+  <section class ="first-wrapper">
+
+    <h1 class="poppins">I'm Victoria Glad to see you!</h1>
+
+    <p class="poppins-regular">
+      I'm a software developer! I can help you build a product, feature 
+      or website. Look through some of my work and experience! If you like 
+      what you see and have a project you need coded, do not hesitate to contact me.
+    </p>
+    <h3 class="poppins-regular">LET'S CONNECT</h3>
+      <ul class="social-media-icons">
+        <li class="a"><img  src="./assets/twitter.svg"></li>
+        <li class="a"><img  src="./assets/Linkedin icon.svg"></li>
+        <li class="a"><img  src="./assets/M.svg"></li>
+        <li class="a"><img  src="./assets/cat.svg"></li>
+        <li class="a"><img  src="./assets/peace.svg"></li>
+      </ul>
+  </section>
+  <br/>
+  <br/>
+
+  <section class="work-section">
```

Comment:
Great use of semantic tags.

Reply:
**[OPTIONAL]:** To make your PRs descriptions and review comments more human-friendly I would recommend using emojis :100: :+1: :technologist: :heavy_check_mark: